cmd/boulder-ra/main.go

```diff
@@ -175,18 +175,27 @@ func main() {
 	cmd.FailOnError(err, "Failed to load credentials and create gRPC connection to Publisher")
 	pubc = bgrpc.NewPublisherClientWrapper(pubPB.NewPublisherClient(conn))
 
+	var groups []cmd.CTGroup
 	if c.RA.CTLogGroups != nil {
```

Comment:
I think we can get rid of the `CTLogGroups` field now, right? We're fully on `2`?